spec/system/ahoy/tracking_create_account_spec.rb

```diff
@@ -13,16 +13,19 @@
       visit root_path
     end
 
-    it "expects page to have the necessary tracking elements", :aggregate_failures do
-      # rubocop:disable RSpec/Capybara/SpecificMatcher
+    it "has the necessary initial tracking elements", :aggregate_failures do
```

Comment:
I this belongs in the Cypress tests, and we can move it away from Capybara, for now I updated it here but when I take a look at best practices we can perhaps look into where it actually belongs